controllers/ingressController.go

```diff
@@ -72,22 +66,17 @@ func WhiteListIP(c *gin.Context) {
 		return
 	}
 	leases = GetActiveLeases(ns, name)
-	for kubeContext, kubeClient := range config.KubeClients {
-		clientset := kubeClient.ClientSet
-		myclientset := pkg.MyClientSet{Clientset: clientset}
-		data, err = myclientset.GetIngress(kubeContext, ns, name)
-		if err != nil {
-			errs = errs + 1
-		}
-		if data.Name != "" {
-			myIngress = data
-			break
-		}
+
+	showIngressDetailsRequest := ingress_driver.ShowIngressDetailsRequest{
```

Comment:
same for `ShowIngressDetailsRequest{}`

Reply:
i didnt fully get your comment but -
this function / code was getting previously called in all the controllers to populate 
![image](https://user-images.githubusercontent.com/5676959/93422513-87e4e600-f8d1-11ea-9197-f875e9a83aed.png)

so i abstracted it into ShowIngressDetails function taking in ShowIngressDetailsRequest and returning ShowIngressDetailsResponse.

was this the question ?